MC/config/QC/json/qc-mft-cluster.json

```diff
@@ -39,7 +39,11 @@
           "query": "randomcluster:MFT/COMPCLUSTERS/0;clustersrof:MFT/CLUSTERSROF/0;patterns:MFT/PATTERNS/0"
         },
         "taskParameters": {
-          "myOwnKey": "myOwnValue"
+          "maxClusterROFSize" : "5000",
+          "maxDuration" : "60000",
+          "timeBinSize" : "0.1",
+          "ROFLengthInBC" : "198",
+          "geomFileName" : "/home/epn/odc/files/o2sim_geometry-aligned.root"
```

Comment:
i am not sure if MC is ran with this file at this location

Reply:
I have removed the full path and left only the file name.